### DISCUSSION TITLE
Bundling Products

### DISCUSSION THREAD

**Discussion by mattrclarke** [vendurehq/vendure#1265] [General]
HI - is currently any concept in vendure of bundled products?
Essentially i'm trying to create a bundle - a single variant, that when purchased - reduces the inventory of the products in that bundle. 
I can do this through grouping variants through facets - 
so when a bundle is bought all the other variants with a matching `bundle_id` facet get added to cart, and then applying a promotion of buy x get all the products in the bundle free, but I feel like there is probably a better way to do this I might not have come across yet? 
Any thoughts would be appreciated.

**Comment by michaelbromley** [comment]
Hi,
There is an open issue for this: https://github.com/vendure-ecommerce/vendure/issues/236 

Basically it's not natively supported right now, but planned for v2. One comment near the end explains how someone implemented it as a plugin though, which might be useful for you.

**Reply by mattrclarke** [reply]
Great, thanks for the quick response
